tests/base64_tests.cpp

```diff
@@ -174,19 +174,115 @@ size_t add_garbage(std::vector<char_type> &v, std::mt19937 &gen,
   std::uniform_int_distribution<int> char_dist(
       0, (1 << (sizeof(char_type) * 8)) - 1);
   uint8_t c = char_dist(gen);
-  while (uint8_t(c) == c && table[uint8_t(c)] != 255) {
+  while ((uint8_t(c) == c && table[uint8_t(c)] != 255) || c == '=') {
```

Comment:
not part of this PR, but the  `(uint8_t(c) == c &&` could be removed

Reply:
Yes, there is a 'bug' there. Fixing.